modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/raft/CmgRaftService.java

```diff
@@ -135,8 +135,8 @@ public CompletableFuture<Void> updateClusterState(ClusterState clusterState) {
      *         otherwise.
      * @see ValidationManager
      */
-    public CompletableFuture<Void> startJoinCluster(ClusterTag clusterTag) {
-        ClusterNodeMessage localNodeMessage = nodeMessage(clusterService.topologyService().localMember());
+    public CompletableFuture<Void> startJoinCluster(ClusterTag clusterTag, String nodeAttributes) {
```

Comment:
Discussed in private, that was made because we want to have only `LogicalNode` in `org.apache.ignite.internal.cluster.management.topology.LogicalTopology`, so we have to pass this attributes for start as well.